packages/contracts-bedrock/src/L1/OptimismPortal2.sol

```diff
@@ -311,7 +311,7 @@ contract OptimismPortal2 is Initializable, ResourceMetering, ISemver {
         // Designate the withdrawalHash as proven by storing the `disputeGameProxy` & `timestamp` in the
         // `provenWithdrawals` mapping. A `withdrawalHash` can only be proven once unless the dispute game it proved
         // against resolves against the favor of the root claim.
-        provenWithdrawals[withdrawalHash] =
+        provenWithdrawals[withdrawalHash][msg.sender] =
```

Comment:
We need to update the `WithdrawalProven` event to also emit the address of the user who proved the withdrawal.

Reply:
Using reverse lookup mapping since `proxyd` isn't a fan of event filtering.